jdaviz/configs/imviz/tests/test_viewers.py

```diff
@@ -16,9 +16,14 @@ def test_create_destroy_viewer(imviz_helper, desired_name, actual_name):
     assert imviz_helper.app.get_viewer_ids() == ['imviz-0']
 
     viewer = imviz_helper.create_image_viewer(viewer_name=desired_name)
+    viewer_names = sorted(['imviz-0', actual_name])
     assert isinstance(viewer, ImvizImageView)
     assert viewer is imviz_helper.app._viewer_store.get(actual_name), list(imviz_helper.app._viewer_store.keys())  # noqa
-    assert imviz_helper.app.get_viewer_ids() == sorted(['imviz-0', actual_name])
+    assert imviz_helper.app.get_viewer_ids() == viewer_names
+
+    # Make sure plugins that store viewer_items differently are consistent.
+    assert imviz_helper.plugins['Imviz Line Profiles (XY)']._obj.viewer_items == viewer_names
+    assert sorted(imviz_helper.plugins['Compass'].viewer.labels) == viewer_names

```

Comment:
This test would fail like this in `main`.

```
_______________________________________ test_create_destroy_viewer[None-imviz-1] _______________________________________

imviz_helper = <jdaviz.configs.imviz.helper.Imviz object at 0x7f68701e4550>, desired_name = None
actual_name = 'imviz-1'

    @pytest.mark.parametrize(
        ('desired_name', 'actual_name'),
        [(None, 'imviz-1'),
         ('babylon-5', 'babylon-5')])
    def test_create_destroy_viewer(imviz_helper, desired_name, actual_name):
        assert imviz_helper.app.get_viewer_ids() == ['imviz-0']

        viewer = imviz_helper.create_image_viewer(viewer_name=desired_name)
        viewer_names = sorted(['imviz-0', actual_name])
        assert isinstance(viewer, ImvizImageView)
        assert viewer is imviz_helper.app._viewer_store.get(actual_name), list(imviz_helper.app._viewer_store.keys())  # noqa
        assert imviz_helper.app.get_viewer_ids() == viewer_names

        # Make sure plugins that store viewer_items differently are consistent.
        assert imviz_helper.plugins['Imviz Line Profiles (XY)']._obj.viewer_items == viewer_names
>       assert sorted(imviz_helper.plugins['Compass'].viewer.labels) == viewer_names
E       AssertionError: assert ['ImvizImageView', 'imviz-0'] == ['imviz-0', 'imviz-1']
E         At index 0 diff: 'ImvizImageView' != 'imviz-0'
E         Use -v to get more diff

jdaviz/configs/imviz/tests/test_viewers.py:26: AssertionError
```